test/test-viz.cpp

```diff
@@ -0,0 +1,311 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <tuple>
+#include <variant>
+
+#include <lorina/bench.hpp>
+
+#define SDL_MAIN_HANDLED
+
+#include <SDL.h>
+
+enum SPACING {
```

Comment:
Unconventional naming: UPPER_SNAKE_CASE is generally used for global constants or macros. Moreover, it is fine to not name a type at all when it's name is not referenced:
```suggestion
enum {
```